### PR TITLE
jQuery Easy Ticker plugin

### PR DESCRIPTION
**Pull request by hjames9**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/vaakash/jquery-easy-ticker/
 * Official website (optional, not the repository): http://www.aakashweb.com/jquery-plugins/easy-ticker/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 106
   - Count of watchers: 15
   - Count of forks: 72
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [X ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [X ] The lib has notable popularity
   * [X ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [X ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jquery-easy-ticker/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "news",
+    "rss",
+    "feed",
+    "vertical",
+    "easing"
+  ],
+  "repository": {
+    "type": "plain file",
+    "url": "https://github.com/vaakash/jquery-easy-ticker/blob/master/jquery.easy-ticker.min.js"
+  },
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-easy-ticker/package.json]: Repository is wrong, please take a look at other libraries like `jqurey`, thanks.

**Comment by PeterDaveHello** [comment]
@hjames9 thanks for your PR, would you please also add auto-update config in the package.json? Thanks.

File: ajax/libs/jquery-easy-ticker/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jquery-easy-ticker",
+  "filename": "jquery.easy-ticker.min.js",
+  "version": "2.0.0",
+  "description": "jQuery Easy ticker plugin",
+  "homepage": "http://www.aakashweb.com/jquery-plugins/easy-ticker/",
```
Comment (kennynaoh) [inline ajax/libs/jquery-easy-ticker/package.json]: I think `https://www.aakashweb.com/jquery-plugins/easy-ticker/` will be better, thanks!

**Comment by hjames9** [comment]
Fixed all the issues mentioned.  Thanks!

File: ajax/libs/jquery-easy-ticker/package.json [hunk 3]
```diff
@@ -18,9 +18,17 @@
     "vertical",
     "easing"
   ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/vaakash/jquery-easy-ticker.git",
+    "basePath": "dist",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-easy-ticker/package.json]: There is no `dist` folder, so it should be empty.

File: ajax/libs/jquery-easy-ticker/package.json [hunk 4]
```diff
@@ -18,9 +18,17 @@
...
+    "files": [
+      "**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-easy-ticker/package.json]: We only need to get `"jquery.easy-ticker*"`, `"**/*"` means all the files.

File: ajax/libs/jquery-easy-ticker/package.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  "description": "jQuery Easy ticker plugin",
+  "homepage": "https://www.aakashweb.com/jquery-plugins/easy-ticker/",
+  "keywords": [
+    "jquery",
+    "ticker",
+    "easy",
+    "animation",
+    "slider",
+    "effect",
+    "ticker",
```
Comment (pvnr0082t) [inline ajax/libs/jquery-easy-ticker/package.json]: The same keyword as line 9. Please remove it, thanks.

Comment (PeterDaveHello) [inline ajax/libs/jquery-easy-ticker/package.json]: Great catch, maybe we could add this in the test!

**Comment by hjames9** [comment]
I hope I got it this time. @PeterDaveHello @pvnr0082t

**Comment by PeterDaveHello** [comment]
Thanks @hjames9

**Pull request merged**